ratatui-core/Cargo.toml

```diff
@@ -46,12 +52,14 @@ serde = ["dep:serde", "bitflags/serde", "compact_str/serde"]
 anstyle = { version = "1", optional = true }
 bitflags = "2.3"
 compact_str = "0.9.0"
+critical-section = "1.2.0"
 document-features = { workspace = true, optional = true }
 hashbrown = "0.15.2"
 indoc.workspace = true
 itertools.workspace = true
 kasuari = "0.4.0"
 lru = "0.12.0"
+once_cell = "1.21.3"
```

Comment:
core has OnceCell at https://doc.rust-lang.org/core/cell/struct.OnceCell.html, so I'd prefer avoiding this dep unless there's a rationale for it.

Reply:
Ah got it - makes sense. Given the proposed alternatives this might go away possibly still, but I see why it is necessary for this approach.